addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -42,25 +42,23 @@
                         <div class="o_wevent_sidebar_block">
                             <h6 class="o_wevent_sidebar_title">Date &amp; Time</h6>
                             <div class="d-flex">
-                                <h5 t-field="event.date_begin" class="my-1 me-1 oe_hide_on_date_edit" t-options="{'tz_name': event.date_tz, 'date_only': 'true', 'format': 'EEEE'}"/>
-                                <h5 class="my-1 oe_hide_on_date_edit" t-field="event.date_begin" t-options="{'tz_name': event.date_tz, 'date_only': 'true', 'format': 'long'}" itemprop="startDate" t-att-datetime="event.date_begin"/>
+                                <h5 t-field="event.date_begin" class="my-1" t-options="{'tz_name': event.date_tz, 'format': 'EEEE MMMM dd, YYYY'}" t-att-datetime="event.date_begin"/>
                             </div>
                             <t t-if="not event.is_one_day">Start -</t>
-                            <span t-field="event.date_begin" t-options="{'tz_name': event.date_tz, 'time_only': 'true', 'format': 'short'}"/>
+                            <span t-out="event.date_begin" t-options="{'widget': 'datetime', 'tz_name': event.date_tz, 'time_only': 'true', 'format': 'short'}"/>
                             <t t-if="event.is_one_day">
                                 <i class="fa fa-long-arrow-right mx-1"/>
-                                <span t-field="event.date_end" t-options="{'tz_name': event.date_tz, 'time_only': 'true', 'format': 'short'}"/>
```

Comment:
Hum I tried to do something since it was a little odd in some case.
So I've extracted the date_end time and depending on `is_one_day`, it's either a `t-field` or a `t-out`